docs/introduction/Cross-chain-transfer-protocol.rst

```diff
@@ -31,7 +42,16 @@ As a consequence of the sidechain declaration command, a unique sidechain id wil
 Forward Transfer
 ================
 
-A forward transfer sends coins from the mainchain to a sidechain. The Horizen Mainchain supports a “Forward Transfer” transaction type that specifies the sidechain destination (*sidechain id* and *receiver address*) and the amount of ZEN to be sent. From a mainchain's perspective, the transferred coins are destroyed;  they are only represented in the total balance of that particular sidechain.
+A forward transfer sends coins from the mainchain to a sidechain. The Horizen Mainchain supports a “Forward Transfer” transaction type that specifies the sidechain destination (*sidechain id* and *receiver address*) and the amount of ZEN to be sent.
+Forward Transfer can be done by using following RPC command:
+
+.. code:: Bash
+
+   sc_send {"address": sc_address, "amount": sc_ft_amount, "scid": scid, "mcReturnAddress": mc_return_address}
+
+The specifies where the forward transfer coins are sent("address"), it's amount("amount"), sidechain destination and address for change("mcReturnAddress").
+
+From a mainchain's perspective, the transferred coins are destroyed;  they are only represented in the total balance of that particular sidechain.
```

Comment:
From a mainchain's perspective, the transferred coins are destroyed; they are only represented in the total balance of that particular sidechain.